tests/js-units/node/config/baselayer.test.js

```diff
@@ -834,7 +834,7 @@ describe('BaseLayersConfig', function () {
             "format": "image/png;%20mode%3D8bit",
             "layers": "commune",
             "styles": "d%C3%A9faut",
-            "url": "https://demo.lizmap.com/lizmap/index.php/lizmap/service?repository=miscellaneous&project=flatgeobuf&VERSION=1.3.0"
+            "url": "https://liz.lizmap.com/tests/index.php/lizmap/service?repository=miscellaneous&project=flatgeobuf&VERSION=1.3.0"
```

Comment:
Arf, thanks.
Yesterday, as I was updating the demo website, I removed the `cypress` folder. Thanks for the update to the liz server.